router/pkg/config/config.go

```diff
@@ -962,7 +962,16 @@ type Config struct {
 	AutomaticPersistedQueries      AutomaticPersistedQueriesConfig `yaml:"automatic_persisted_queries"`
 	ApolloCompatibilityFlags       ApolloCompatibilityFlags        `yaml:"apollo_compatibility_flags"`
 	ApolloRouterCompatibilityFlags ApolloRouterCompatibilityFlags  `yaml:"apollo_router_compatibility_flags"`
-	ClientHeader                   ClientHeader                    `yaml:"client_header"`
+
+	Extensions   ExtensionsConfig `yaml:"extensions"`
+	ClientHeader ClientHeader     `yaml:"client_header"`
+}
+
+type ExtensionsConfig struct {
```

Comment:
Q: do you have a global option to fully skip extensions?

Reply:
No, because that would mean that I need to find every place which creates/prints an `Extension` node. As this is more about whitelisting (or rather skipping extensions as needed) I skipped adding this.